mail/opensmtpd/Portfile

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        opensmtpd opensmtpd 6.8.0p1-rc1
```

Comment:
We would want to use a stable version, not a release candidate.

Reply:
Right. 6.8.0p1 will be released this month, so I'll wait until it get released before turning this draft pull request into an actual pull request (I also didn't think that draft pull requests were publicly visible!).